package.json

```diff
@@ -13,7 +13,7 @@
     "relay2ts": "relay2ts src/components/*.tsx --update",
     "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/master && yarn install && npm run dump-schema -- ../../data",
     "sync-colors": "cd externals/elan && git fetch && git checkout origin/master && cp components/lib/variables/colors.json ../../data",
-    "lint": "tslint -c tslint.json --project tsconfig.json",
+    "lint": "tslint -c tslint.json --project tsconfig.json --fix",
```

Comment:
I don’t think we should do this, it would also auto-fix fails on CI (but not commit them) and be pseudo-green.

Instead you can pass the flag when you need it: `yarn lint -- --fix`